chainer/utils/conv.py

```diff
@@ -13,6 +31,23 @@ def get_conv_outsize(size, k, s, p, cover_all=False, d=1):
 
 
 def get_deconv_outsize(size, k, s, p, cover_all=False):
+    """Calculate output size of deconvolution.
+
+    This functions takes the size of input feature map, kernel, stride, and
+    pooling, then calculate the output feature map size.
```

Comment:
I think it's better to explicitly mention that this function only calculates for a single dimension.

Reply:
What do you mean? Could you suggest some concrete sentences?

Reply:
Something like this:

"This function takes the size of input feature map, kernel, stride, and pooling of one particular dimension, then calculates the output feature map size of that dimension. "

Reply:
Thanks! I used your words.